src/main/java/commands/ToDoCommand.java

```diff
@@ -0,0 +1,30 @@
+package commands;
+
+import data.Task;
+import data.TaskList;
+import data.ToDo;
+import exceptions.DukeException;
+import storage.Storage;
+import ui.Ui;
+
+public class ToDoCommand extends Command {
+    private final String description;
+
+    public ToDoCommand(String description) throws DukeException {
+        if (description.length() == 0) {
+            throw new DukeException("☹ OOPS!!! The description of a todo cannot be empty.");
+        }
```

Comment:
There is a similar check before the only call of the ToDoCommand constructor. Perhaps either that check or this second check could be omitted?